glossary.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Glossary
+nav_order: 7
+published: false
+---
+# Glossary of Common Terms
+
+This page contains a selection of key terms important to understanding the structure of data within Data Commons.
+
+## Term List
+
+- **DCID**
+
+A unique identifier itemizing and classifying an entity in the Data Commons graph. For example, Austin, Texas, has a DCID of 'geoId/4805000', while the plant species _Austrobaileya scandens_ has a DCID of 'dc/bsmvthtq89217'.
+
+- **[Statistical variable](https://datacommons.org/browser/StatisticalVariable)**
+
+Any type of metric, statistic, or measure that can be measured at a place and time. Examples include [median income of persons older than 16](https://datacommons.org/browser/Median_Income_Person_16OrMoreYears), [number of female high school graduates aged 18 to 24](https://datacommons.org/browser/Count_Person_18To24Years_EducationalAttainmentHighSchoolGraduateIncludesEquivalency_Female), [unemployment rate](https://browser.datacommons.org/browser/UnemploymentRate_Person), or [percentage of persons with diabetes](https://browser.datacommons.org/browser/Percent_Person_WithDiabetes). A complete list of statistical variables can be found at <https://datacommons.org/browser/StatisticalVariable>.
```

Comment:
While less complete, it could be good to refer to https://docs.datacommons.org/statistical_variables.html somewhere here for users